bridgetown-plausible.gemspec

```diff
@@ -8,7 +8,7 @@ Gem::Specification.new do |spec|
   spec.authors = ["Andrew Mason"]
   spec.email = ["andrewmcodes@protonmail.com"]
   spec.summary = "Plausible Analytics Plugin for Bridgetown"
-  spec.description = "A Liquid tag to add Plausible analytics to your site."
+  spec.description = "A Liquid/erb tag to add Plausible analytics to your site."
```

Comment:
**nitpick:** To avoid confusion for folks trying to understand the difference between Liquid tags and ERB helpers, I want to make sure we are using the correct terminology per the Bridgetown docs.